CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Version 1.2.3 (20SEP22)
+
+* **Overall**
+  * [FEATURE] - The V4M container now includes OpenSSL so that you can run the deployment with TLS.
+  * [BUG] - Adjusted OpenSSL command that created an unnecssary file that caused issues if multiple users tried running the deployment scripts on the same machine.
```

Comment:
Sorry, I described the temp file issue in the "logging" section but it should be up here in the Overall.  So, you can move the item under "logging" up to here and drop this one (since it's the same change).